linalg/hypre.cpp

```diff
@@ -1018,6 +1030,47 @@ HypreParMatrix * HypreParMatrix::Transpose() const
    return new HypreParMatrix(At);
 }
 
+#if MFEM_HYPRE_VERSION >= 21800
+HypreParMatrix * HypreParMatrix::ExtractSubmatrix(Array<int> &indices,
+                                                  double threshhold) const
+{
+   if (!(A->comm))
+   {
+      BuildComm();
+   }
+
+   hypre_ParCSRMatrix *submat;
+
+   // Get number of rows stored on this processor
+   int local_num_vars = hypre_CSRMatrixNumRows(hypre_ParCSRMatrixDiag(A));
+
+   // Form hypre CF-splitting array designating submatrix as F-points (-1)
+   int *CF_marker = new int[local_num_vars];
+   std::fill_n(CF_marker, local_num_vars, 1);
```

Comment:
```suggestion
   Array<int> CF_marker(local_num_vars);
   CF_marker = 1;
```

Reply:
If CF_marker is an MFEM Array, is it passes as a pointer to hypre_ParCSRMatrixExtractSubmatrixFC?

Reply:
Yes, MFEM's `Array<T>` has an implicit cast to a pointer of type `T*`.